Makefile

```diff
@@ -1,9 +1,53 @@
+SHELL = /bin/bash
 # We assume an active virtualenv for development
-install:
-	-@python3 setup.py -q develop --upgrade
+include requirements.txt
+PYENV_LOCATION = /usr/local/bin/pyenv
```

Comment:
This is operating system/distribution specific.

Reply:
Removed.